cpc/model.py

```diff
@@ -104,6 +104,206 @@ def forward(self, x):
         x = F.relu(self.batchNorm4(self.conv4(x)))
         return x
 
+class Smartpool(nn.Module):
```

Comment:
Remove Smartpool, DoXTimes and CPCSmartpool encoder

Reply:
done